nomos-services/data-availability/verifier/src/backend/kzgrs.rs

```diff
@@ -30,7 +31,22 @@ impl VerifierBackend for KzgrsDaVerifier {
     type Settings = KzgrsDaVerifierSettings;
 
     fn new(settings: Self::Settings) -> Self {
-        let verifier = NomosKzgrsVerifier::new(settings.sk, &settings.nodes_public_keys);
+        let bytes = hex::decode(settings.sk).expect("Secret key string should decode to bytes");
+        let secret_key =
+            SecretKey::from_bytes(&bytes).expect("Secret key should be reconstructed from bytes");
+
+        let nodes_public_keys = settings
+            .nodes_public_keys
+            .iter()
+            .map(|pk_hex| {
+                let pk_bytes =
+                    hex::decode(pk_hex).expect("Public key string should decode to bytes");
+                PublicKey::from_bytes(&pk_bytes)
+                    .expect("Public key should be reconstructed from bytes")
+            })
+            .collect::<Vec<PublicKey>>();
+
+        let verifier = NomosKzgrsVerifier::new(secret_key, &nodes_public_keys);
```

Comment:
I think this is not needed anymore. But i have to check on it.

Reply:
I've seen that sk is not used anymore, but the index is still required. Lets do it in a separate PR.